src/pages/AppDetail.svelte

```diff
@@ -53,7 +54,10 @@
     schema={APPLICATION_DEFINITION_SCHEMA}
   />
 
-  <h2>Commentary</h2>
+  <SubHeading
+    title={"Commentary"}
+    helpText={"Reviewed commentary from Mozilla data practitioners on this application."}
```

Comment:
There's a bunch of duplication here but I'd rather leave fixing that to a separate PR